docs/howto/clean_old_jobs.rst

```diff
@@ -0,0 +1,25 @@
+.. _clean_old_jobs:
+
+Clean old jobs
+--------------
+
+You may want to clean your database by removing old jobs. Procrastinate exposes a builtin
+task which lets you do just that. Note that jobs and linked events will be irreversibly removed
+from the database when running this task.
+
+From the CLI
+^^^^^^^^^^^^
+
+.. code-block:: console
+
+    $ procrastinate defer procrastinate.builtin_tasks.remove_old_jobs max_hours=72 remove_error=true
+
+The task takes 2 paremeters related to which jobs to delete:
+
+- ``max_hours``: Jobs which were created more than ``max_hours`` ago will be deleted
```

Comment:
Hmm... Why are we filtering on the creation date and not the completion date ? It means a 3 days old job that runs just now might be deleted right after running... I'm not sure it's what we want...

Reply:
I can change this :+1: